.luacheckrc

```diff
@@ -0,0 +1,105 @@
+codes = true
+std = "min"
+self = false
+
+read_globals = {
+	"getfenv",
+	"setfenv",
+	"unpack",
+}
+
+include_files = {
+	"**/*.lua",
+	"package/**/files/lib/gluon/ebtables/*",
```

Comment:
I just moved the ebtables files to luasrc, so we can get rid of this rule. The extra rule for ebtables below needs to be adjusted as well.

Reply:
Dropped and adjusted